language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/LSAnnotationCache.java

```diff
@@ -54,26 +57,27 @@
 
     private static final Logger logger = LoggerFactory.getLogger(LSPackageCache.class);
 
-    private static HashMap<PackageID, List<BAnnotationSymbol>> typeAnnotations = new HashMap<>();
-    private static HashMap<PackageID, List<BAnnotationSymbol>> classAnnotations = new HashMap<>();
-    private static HashMap<PackageID, List<BAnnotationSymbol>> objectAnnotations = new HashMap<>();
-    private static HashMap<PackageID, List<BAnnotationSymbol>> functionAnnotations = new HashMap<>();
-    private static HashMap<PackageID, List<BAnnotationSymbol>> objectMethodAnnotations = new HashMap<>();
-    private static HashMap<PackageID, List<BAnnotationSymbol>> resourceAnnotations = new HashMap<>();
-    private static HashMap<PackageID, List<BAnnotationSymbol>> parameterAnnotations = new HashMap<>();
-    private static HashMap<PackageID, List<BAnnotationSymbol>> returnAnnotations = new HashMap<>();
-    private static HashMap<PackageID, List<BAnnotationSymbol>> serviceAnnotations = new HashMap<>();
-    private static HashMap<PackageID, List<BAnnotationSymbol>> listenerAnnotations = new HashMap<>();
-    private static HashMap<PackageID, List<BAnnotationSymbol>> annotationAnnotations = new HashMap<>();
-    private static HashMap<PackageID, List<BAnnotationSymbol>> externalAnnotations = new HashMap<>();
-    private static HashMap<PackageID, List<BAnnotationSymbol>> varAnnotations = new HashMap<>();
-    private static HashMap<PackageID, List<BAnnotationSymbol>> constAnnotations = new HashMap<>();
-    private static HashMap<PackageID, List<BAnnotationSymbol>> workerAnnotations = new HashMap<>();
-    private static HashMap<PackageID, List<BAnnotationSymbol>> fieldAnnotations = new HashMap<>();
-    private static HashMap<PackageID, List<BAnnotationSymbol>> recordFieldAnnotations = new HashMap<>();
-    private static HashMap<PackageID, List<BAnnotationSymbol>> objectFieldAnnotations = new HashMap<>();
+    private static final HashMap<ModuleID, List<AnnotationSymbol>> typeAnnotations = new HashMap<>();
```

Comment:
Shall we use `Map` interface here? `Map<ModuleID, List<AnnotationSymbol>> typeAnnotations = new HashMap<>()`

Reply:
fixed